lib/secrets.rb

```diff
@@ -44,6 +48,8 @@ def write_secrets
       File.write("#{@output_path}/#{key}", contents)
       puts "Writing #{key} with contents from secret key #{path}"
     end
+    # Write out the pod's status.hostIP as a secret
+    File.write("#{@output_path}/HOST_IP", hostip)
```

Comment:
it's called hostIP in the pod definition, trying to stay close to what they have there.